gengokit/httptransport/templates.go

```diff
@@ -46,6 +46,8 @@ var ClientEncodeTemplate = `
 	// the http request (path, query, and body).
 	func EncodeHTTP{{$binding.Label}}Request(_ context.Context, r *http.Request, request interface{}) error {
 		fmt.Printf("Encoding request %v\n", request)
+		strval := ""
+		_ = strval
```

Comment:
The `_ = strval` is here for cases where there are no path parameters to be parsed, if there are no path parameters to be parsed then just having `strva := ""` would cause a compiler error as it is unused.